src/gc/gcconfig.h

```diff
@@ -64,6 +70,7 @@ class GCConfigStringHolder
 // Each one of these keys produces a method on GCConfig with the name "Get{name}", where {name}
 // is the first parameter of the *_CONFIG macros below.
 #define GC_CONFIGURATION_KEYS \
+    BOOL_CONFIG(AllowVeryLargeObjects, "gcAllowVeryLargeObjects", true, "Allow allocation of 2GB+ objects on GC heap") \
```

Comment:
>AllowVeryLargeObjects [](start = 16, length = 21)

why do we need this? GC does not use it.

Reply:
It would be best to delete this (in this or separate change). `gcAllowVeryLargeObjects` was a desktop compact quirk that serves no real purpose in .NET Core.

Reply:
SGTM